packages/nextjs/utils/scaffold-eth/contract.ts

```diff
@@ -232,6 +233,7 @@ export type UseScaffoldEventHistoryConfig<
   blockData?: TBlockData;
   transactionData?: TTransactionData;
   receiptData?: TReceiptData;
+  watch?: TWatch;
```

Comment:
it seems just `boolean` works fine here

Reply:
I defined a type to be similar to the other parameters:

`  TBlockData extends boolean = false, TTransactionData extends boolean = false, TReceiptData extends boolean = false,`

Reply:
yes, but other params are used later in `UseScaffoldEventHistoryData` type

Reply:
You are right! Changed! Thanks!!